src/swift/flwr/Sources/Flower/Common/Typing.swift

```diff
@@ -45,11 +65,13 @@ public struct FitRes {
     public var parameters: Parameters
     public var numExamples: Int
     public var metrics: Metrics? = nil
+    public var status: Status
     
     public init(parameters: Parameters, numExamples: Int, metrics: Metrics? = nil) {
         self.parameters = parameters
         self.numExamples = numExamples
         self.metrics = metrics
+        self.status = Status(code: .ok, message: String())
```

Comment:
The status should be left as a parameter

`public init(status: Status, parameters: Parameters, numExamples: Int, metrics: Metrics? = nil)`